components/arm/arm.go

```diff
@@ -5,15 +5,12 @@ package arm
 
 import (
 	"context"
-	"errors"
 	"fmt"
-	"strings"
 
 	v1 "go.viam.com/api/common/v1"
 	pb "go.viam.com/api/component/arm/v1"
 
 	"go.viam.com/rdk/data"
-	"go.viam.com/rdk/logging"
 	"go.viam.com/rdk/motionplan"
```

Comment:
what's still pulling this in?

Reply:
Is it the Status function's ComputeOOBPosition? If so we can do away with that since the Status API is being deprecated wholesale at some point.

Reply:
Also very okay if your changes address nlopt fully, I was just expecting the dependency to be pulled in from this particular package.

Reply:
Pretty sure this does also need to be removed, yes

Reply:
We need `ComputeOOBPosition` but IMO it should live in `referenceframe`, not `motionplan`

Reply:
Oops I missed this, will fix